src/zope/__init__.py

```diff
@@ -1,7 +1,7 @@
 # this is a namespace package
 try:
-    import pkg_resources
-    pkg_resources.declare_namespace(__name__)
+    from pkg_resources import declare_namespace
+    declare_namespace(__name__)
 except ImportError:
     import pkgutil
     __path__ = pkgutil.extend_path(__path__, __name__)
```

Comment:
So, AFAIU the `zope/__init__.py` is supposed to be identical in all `zope.*` packages (because on Python 2 you have no control which one ends up being installed), so if this one is wrong, we'll have to do a mass-update.